src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+@use './variables' as *;
+
+@mixin oppsite($width) {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  box-sizing: border-box;
+  width: $paragraph-width;
```

Comment:
The variable `$paragraph-width` is used here. Ensure that this variable is defined in the `variables` file or elsewhere in the project. If it's meant to be a parameter, consider using the `$width` parameter instead.